src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/EntityConnectionExport.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types = 1);
+
+/**
+ * Copyright 2024 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Domain\ValueObject;
```

Comment:
Isn't it the characteristic of a ValueObject to be immutable and self validating? This is more a Dto?

Reply:
In my opinion it is not really transfering any domains. But I agree it is not a a pure VO given the mutability. If you dont mind to much. Im leaving this as is